.env.dist

```diff
@@ -1,8 +1,12 @@
 TEAM_MODEL_ID = "replace_with_team_model_google_sheet_id"
-SUPPORT_SCHEDULER_HISTORY_ID = "replace_with_Support_Scheduler_History_google_sheet_id"
-CALENDAR_ID="replace_with_Support_schedule_google_calendar_id"
 GAPI_SERVICE_ACCOUNT_JWT="replace_with_path_to_google_json_credentials"
 SUPPORT_RESOURCES="replace_with_link_to_support_process_documentation"
-CREDENTIALS="replace_with_path_to_google_json_credentials (for your g-api app)"
-TOKEN="replace_with_link_to_your_token"
-PRODUCT_OS_CALENDAR_ID="replace_with_productOS_schedule_google_calendar_id"
\ No newline at end of file
+
+# Add your google oauth app credentials into creds.json
+CREDENTIALS="./creds.json"
+# Your oauth token will be stored in token.json once you have generated it
+TOKEN="./token.json"
```

Comment:
Does it make sense to name it `G_CREDENTIALS` and `G_TOKEN` to signify that these are Google-related?
Similar to how `VO` stands for VictorOps below.